utils/clone-remote-to-exercise.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+clone_remote_to_exercise() {
+    rm -rf exercise/
+
+    # Clone remote
+    git clone ./remote exercise
```

Comment:
`./remote` -> `remote` will give slightly nicer paths with `git remote -v`.

Reply:
I will merge the pull request as is - then we can fix this in an iteration. :)